src/client.ts

```diff
@@ -24,6 +24,19 @@ import {
 
 import * as which from 'which'
 
+const VarRegex = new RegExp(/\$\{(.+?)\}/g);
```

Comment:
Can this be changed to `new RegExp(/\$\{(\w+)\}/g)`?

Reply:
Yes, change applied.